src/app/onto-search/onto-search.component.html

```diff
@@ -0,0 +1,11 @@
+<p>onto-search works!</p>
+<app-search
+  [states]="states"
+  [workingTemplate]="customTemplate ? customTemplate : defaultTemplate"
+  (onSearch)="onSearchEvent($event)"></app-search>
+
+  <ng-template #defaultTemplate let-state>
```

Comment:
If I use a customTemplate, can I still bind to the state?

Reply:
Yes, and as it is the $implicit state, it doesn't have to be called state